Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -924,6 +957,38 @@ void MenuBar::UpdateToolsMenu(bool emulation_started)
   }
 }
 
+#if defined(_WIN32)
+#include <Common/FileUtil.h>
```

Comment:
This include isn't necessary, since it's already included at the top of the file.